semeio/jobs/rft/gendata_rft.py

```diff
@@ -158,8 +162,7 @@ def run(
             # Aggregate dataframe for all wells and report steps.
             dframes.append(trajectory_df)
 
-    if csvfile is not None:
-        pd.concat(dframes, ignore_index=True, sort=False).to_csv(csvfile, index=None)
+    pd.concat(dframes, ignore_index=True, sort=False).to_csv(csvfile, index=None)
```

Comment:
If no `csv` file is given, doesnt this write to `stdout`? If so, that fills the `stdout` files with some unwanted text.

Reply:
Yes. Should perhaps reinstate that `if`.